src/ted2zim/entrypoint.py

```diff
@@ -9,6 +9,11 @@ def main():
         description="Scraper to create ZIM files from TED talks topics or playlists",
     )
 
+    parser.add_argument(
+        "--link",
+        help="Comma-seperated list of links to scrape",
```

Comment:
sep**a**rated

Reply:
mention this is the link to the video online, like `https://www.ted.com/talks/<talk_slug>`